main.tf

```diff
@@ -135,7 +135,11 @@ resource "aws_lambda_function" "this" {
     }
   }
 
-  tags = merge(var.tags, var.function_tags)
+  tags = merge(
```

Comment:
@bryantbiggs WDYT about tags? (the rest of the renaming was automatic)

Reply:
The function def, there won't be any insight into the s3 object, cloudwatch could technically be derived but ok either way. Typically, the fewer the better, and tag the main thing which other things can be derived off of that

Reply:
Updated in 16b854a